docs/pages/router/static-rendering/index.mdx

```diff
@@ -0,0 +1,222 @@
+            <meta name="viewport" content="width=device-width, initial-scale=1.0, viewport-fit=cover" />
+        */}
+        <meta
+          name="viewport"
+          content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1.00001,viewport-fit=cover"
+        />
+        {/* 
+          Disable body scrolling on web. This makes ScrollView components work closer to how they do on native. 
+          However, body scrolling is often nice to have for mobile web. If you want to enable it, remove this line.
+        */}
+        <ScrollViewStyleReset />
+
+        {/* Add any additional <head> elements that you want globally available on web... */}
+      </head>
+      <body>{children}</body>
+    </html>
+  );
+}
+```
+
+- The `children` prop comes with the root `<div id="root" />` tag included inside.
+- The JavaScript scripts are appended after the static render.
+- React Native web styles are statically injected automatically.
+- Global CSS should not be imported in this file, instead use the [Root Layout](/router/advanced/root-layout) component.
+- Browser APIs like `window.location` are not available in this component as it only runs in Node.js during static rendering.
+
+### `expo-router/html`
+
+The exports from `expo-router/html` are related to the Root HTML component.
+
+- `ScrollViewStyleReset`: Root style-reset for full-screen React Native web apps with a root `<ScrollView />` should use the following styles to ensure native parity. [Learn more](https://necolas.github.io/react-native-web/docs/setup/#root-element).
+
+## Meta Tags
+
+You can add meta tags to your pages with the [`<Head />`](/docs/features/head) module from `expo-router`:
+
+```js app/about.tsx
+import Head from 'expo-router/head';
+import { Text } from 'react-native';
+
+export default function Page() {
+  return (
+    <>
+      <Head>
+        <title>My Blog Website</title>
+        <meta name="description" content="This is my blog." />
+      </Head>
+      <Text>About my blog</Text>
+    </>
+  );
+}
+```
+
+The head elements can be updated dynamically using the same API, but it's useful for SEO to have static head elements rendered ahead of time.
+
+## Static Files
+
+Expo CLI supports a root `/public` directory that will be copied to the `dist` folder during static rendering. This is useful for adding static files like images, fonts, and other assets.
+
+<FileTree
+  files={['public/favicon.ico', 'public/logo.png', 'public/.well-known/apple-app-site-association']}
+/>
+
+These files will be copied to the `dist` folder during static rendering:
+
+<FileTree
+  files={[
+    'dist/index.html',
+    'dist/favicon.ico',
+    'dist/logo.png',
+    'dist/.well-known/apple-app-site-association',
+    'dist/_expo/static/js/index-xxx.js',
+    'dist/_expo/static/css/index-xxx.css',
+  ]}
+/>
+
+Static assets can be accessed in runtime code using relative paths (**WEB ONLY**). For example, the `logo.png` file can be accessed at `/logo.png`:
```

Comment:
```suggestion
> **info** **Web only**: Static assets can be accessed in runtime code using relative paths. For example, the **logo.png** can be accessed at `/logo.png`:
```

May be this can be a callout